src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -267,3 +267,5 @@ interfaces:
     request_object_method: false
     field_name_patterns:
       name: archived_book
+    sample_code_init_fields:
+      - comment_map{"comment_key"}.comment="default comment"
```

Comment:
Maybe do separate tests for map key and default value instead of testing them in a single field.